.github/workflows/manual_update_development.yml

```diff
@@ -43,4 +43,4 @@ jobs:
         id: link-report
         uses: docker://ghcr.io/threefoldfoundation/website-link-checker:latest
         with:
-          args: 'https://www2.manual.grid.tf -e 404 500 501 502 503 504 -w all'
```

Comment:
Why were 500 and 502 removed?

Reply:
For example, presearch.com triggered 502 error, but if you try it yourself on the manual, the link works.
In practice, only the 404 errors don't work.

Reply:
HTML errors set as warnings (-w) on the link checker exit with 0.
HTML errors set as errors (-e) on the link checker exit with 1.
Personally I even think we should have: -e 404 -w all.
Because, as I understand, the 5** errors work from the user perspective using the manual (the links work).